Packs/GoogleCloudSCC/Integrations/GoogleCloudSCC/GoogleCloudSCC.py

```diff
@@ -0,0 +1,1125 @@
+        body = assign_params(returnImmediately=ret_immediately, maxMessages=max_messages)
+        request = self.service.projects().subscriptions().pull(  # pylint: disable=E1101
+            subscription=subscription, body=body)
+        result = self.execute_request(request)
+        return result
+
+    def acknowledge_messages(self, acks_list: List) -> Dict[str, Any]:
+        """
+        Pull messages for the subscription
+
+        :param acks_list: The maximum number of messages to return for this request. Must be a positive integer
+        :return:
+        """
+        subscription = GoogleNameParser.get_subscription_path(self.project_id, self.subscription_id)
+        body = assign_params(ackIds=acks_list)
+        request = self.service.projects().subscriptions().acknowledge(  # pylint: disable=E1101
+            subscription=subscription, body=body)
+        result = self.execute_request(request)
+        return result
+
+
+""" HELPER FUNCTIONS """
+
+
+def init_google_scc_client(**kwargs) -> GoogleSccClient:
+    """
+    Initializes google scc client
+    :param kwargs: keyword arguments
+    :return: SCC Client object
+    """
+    client = GoogleSccClient(service_name=SERVICE_NAME, service_version=SERVICE_VERSION, scopes=SCOPES, **kwargs)
+    return client
+
+
+def init_google_pubsub_client(**kwargs) -> GooglePubSubClient:
+    """
+    Initializes google scc client
+    :param kwargs: keyword arguments
+    :return: SCC Client object
+    """
+    client = GooglePubSubClient(service_name=PUBSUB_SERVICE_NAME, service_version=PUBSUB_SERVICE_VERSION,
+                                scopes=SCOPES, **kwargs)
+    return client
+
+
+def safe_load_non_strict_json(json_string: str) -> Dict[str, Any]:
+    """
+    Loads the JSON with non-strict mode.
+
+    :param json_string: json string to parse.
+
+    :return: Parsed dictionary.
+    :raises ValueError: If there is any other issues while parsing json.
+    """
+    try:
+        if json_string:
+            return json.loads(json_string, strict=False)
+        return {}
+    except ValueError:
+        raise ValueError(ERROR_MESSAGES["JSON_PARSE_ERROR"].format("Service Account JSON"))
+
+
+def validate_get_int(max_results: Optional[str], message: str, limit: Union[int, str] = 0) -> Optional[int]:
+    """
+    Validate and convert string max_results to integer.
+
+    :param max_results: max results in string.
+    :param message: Message to display when exception raised.
+    :param limit: If max_results > limit raise the exception.
+
+    :return: int max_results
+    :raises ValueError: if max_results is not a integer and < 0.
+    """
+    if max_results:
+        try:
+            max_results_int = int(max_results)
+            if max_results_int <= 0:
+                raise ValueError
+            if limit and max_results_int > int(limit):
+                raise ValueError
```

Comment:
always raise an exception object, not a class
```suggestion
                raise ValueError(message)
            if limit and max_results_int > int(limit):
                raise ValueError(message)
```

Reply:
Done